README.md

```diff
@@ -39,156 +33,11 @@ repository which you can use for reference.
-
-By default, the images are given tags like, say,
-`chronicle-manufacturing-stl-release:local`. A value other than `local` can
-be set in the `ISOLATION_ID` environment variable prior to build.
-
-## Generate the GraphQL Schema
-
-This command uses the env var ISOLATION_ID to set the image tag.
-This defaults to `local` if not set. In our example we will set it to `1.0.0`.
-
-```bash
-export ISOLATION_ID=1.0.0
-
-## GraphQL Client
-
-Chronicle Examples use Chronicle's `serve-graphql` function to provide the
-Chronicle GraphQL API. By using a GraphQL client, you can interact with Chronicle
-by running GraphQL queries, mutations, and subscriptions.
-
-We recommend using the [Altair GraphQL Client](https://altairgraphql.dev/),
-which is available as a free desktop GraphQL IDE or web browser extension.
-
-If you have previously used Chronicle Examples, you can still access the
-[GraphQL Playground](https://github.com/graphql/graphql-playground) through your
-web browser at <http://127.0.0.1:9982>, however we will be deprecating support
-for GraphQL Playground in future releases.
-
-Both of these GraphQL clients are persistent via cookies, so running the same
-browser on the same machine will remember all your queries and tab positions.
-
-To add a new mutation or query tab, there is a `+` on the right-hand side of the
-tab bar.
-
-Once you get to this point, you are ready to explore the example. To do this,
-refer to the relevant guide.
-
-### Notes
-
-If you are using Chronicle on default settings, point the GraphQL client to
-<http://127.0.0.1:9982>.
-
-The *SCHEMA* and *DOCS* tabs are useful for showing the relationship between
-your `domain.yaml` configuration and the resulting Chronicle API.
-
-Shift-refresh on the playground will remove previous results from query tabs,
-which is recommended before rerunning your example.
-
-### Subscribe to Events
-
-Finally, to see what is happening as you run GraphQL mutations and queries, you
-can subscribe to events in one of the tabs by using the subscription URL
-<ws://localhost:9982/ws>.
-
-```graphql
-subscription {
-  commitNotifications {
-    stage
-    error
-    delta
-  }
-}
-```
-
-This command will produce a `chronicle-manufacturing-stl-release:1.0.0`
-docker image.
-
-You will need to push this image to a docker registry, and specify it in
-your Chronicle deployment.
-
-See the following Docker documentation for more details:
+From there, we suggest using the following sections of the Chronicle Examples `README`
+as a guide:
 
-* [Docker registry docs](https://docs.docker.com/registry/)
-* [Docker tag docs](https://docs.docker.com/engine/reference/commandline/tag/)
-* [Docker push docs](https://docs.docker.com/engine/reference/commandline/push/).
+- [Prerequisites](https://github.com/btpworks/chronicle-examples/blob/main/README.md#prerequisites)
+- [Building a Domain](https://github.com/btpworks/chronicle-examples/blob/main/README.md#prerequisites)
```

Comment:
The anchors in these links need updating.

Reply:
Oops! Thanks!